9/Dockerfile

```diff
@@ -63,11 +64,26 @@ RUN mkdir -p /var/run/redis && \
     greenbone-nvt-sync && \
     greenbone-scapdata-sync && \
     greenbone-certdata-sync && \
-    BUILD=true /start && \
+    BUILD=true /start && \ 
     service openvas-scanner stop && \
     service openvas-manager stop && \
     service openvas-gsa stop && \
     service redis-server stop
 
+RUN apt-get update && \
+    apt-get install -y postfix supervisor libsasl2-modules syslog-ng-core mailutils && \
+    apt-get autoremove -yq
+
+COPY sasl_passwd_template /
+COPY main.cf_template /
+
+ENV BUILD=""
+
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
```

Comment:
@mikesplain without dumb-init there is the pid1 issue with this container. Meaning you can not ctrl-c out of the container while its running. 

This also means that when you do a docker stop on the container docker ends up killing the container because it does not exit cleanly. Dumb-init solves this issue.